inference-api/python_client/client.py

```diff
@@ -0,0 +1,112 @@
+import os
+import requests
+
+from asgiref.sync import async_to_sync
+from retry import retry
+from typing import Final, List, Optional
+
+from .exceptions import (
+  FailedToWriteSegmentationError,
+  InvalidInferenceAPIClientArgsException,
+  SegmentationDownloadError,
+)
+from .types import Inference
+from .django_jwt_client import DjangoJWTClient
+
+class InferenceAPIClient:
```

Comment:
Since this repository is intended for external users I've been thoroughly documenting the class and method definitions. See an example here: https://github.com/bunkerhillhealth/bunkerhill-sdk/blob/621c020902260e93c289e47ca51bb0f5dbd5b069/bunkerhill/model_runner.py#L22

I've been following these guidelines dictating docstring style: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings